### DISCUSSION TITLE
每次重启都IDEA都要手动输入密码

### DISCUSSION THREAD

**Discussion by zfovikqdev928** [YiiGuxing/TranslationPlugin#2311] [Q&A] [answered]
如图：每次重启后，都需要重新输入这个密码：![image](https://user-images.githubusercontent.com/66060529/197464210-3657c4fb-090a-45ba-9df3-479da4c33b06.png)
能否解决一下这个问题

**Comment by YiiGuxing** [comment, marked as answer]
可以尝试将密码保存方式改成In KeePass方式 (设置 > 外观和行为 > 系统 设置 > 密码)

![image](https://user-images.githubusercontent.com/10737066/197465139-54440e9b-0d86-49e3-b2e2-e9de1d1ad34a.png)

**Reply by zfovikqdev928** [reply]
感谢，已解决，效果很好使！棒棒棒
